templates/dashboard.jinja.html

```diff
@@ -488,9 +488,6 @@ <h2 class="downloads-header">Downloads</h2>
         Report generated with
         <a href="https://github.com/awslabs/aws-build-accumulator">Litani</a>
         {{ litani_version }}
-        {% if litani_hash is not none %}
```

Comment:
I see you're making the "litani_hash" optional.

If I desire for one to get rendered, can that still be done? What would I need to do differently?

Reply:
It's not optional. I removed it completely to make sure Litani does not shell out to git at all.

Reply:
Sorry, misinterpreted this as addition of lines.